packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -283,79 +535,95 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/crd
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/requirements.yaml packages/rancher-monitoring/charts/requirements.yaml
 --- packages/rancher-monitoring/charts-original/requirements.yaml
 +++ packages/rancher-monitoring/charts/requirements.yaml
-@@ -14,3 +14,98 @@
-     version: "5.3.*"
+@@ -1,16 +1,111 @@
+ dependencies:
+ 
+   - name: kube-state-metrics
+-    version: "2.8.*"
++    version: "2.8.14"
+     repository: https://kubernetes-charts.storage.googleapis.com/
+     condition: kubeStateMetrics.enabled
+ 
+   - name: prometheus-node-exporter
+-    version: "1.10.*"
++    version: "1.10.0"
+     repository: https://kubernetes-charts.storage.googleapis.com/
+     condition: nodeExporter.enabled
+ 
+   - name: grafana
+-    version: "5.3.*"
++    version: "5.3.6"
      repository: https://kubernetes-charts.storage.googleapis.com/
      condition: grafana.enabled
 +
 +  - name: prometheus-adapter
-+    version: 2.4.*
++    version: "2.4.0"
 +    repository: https://kubernetes-charts.storage.googleapis.com/
 +    condition: prometheus-adapter.enabled
 +
 +  - name: rancher-pushprox
 +    alias: rkeControllerManager
-+    version: 0.1.0
++    version: "0.1.0"
```

Comment:
Can these formatting changes be avoided in the patch?

Reply:
yeah sure, will push a commit out